ring_buffer/ring_buffer.py

```diff
@@ -5,7 +5,12 @@ def __init__(self, capacity):
     self.storage = [None]*capacity
 
   def append(self, item):
-    pass
+    self.storage[self.current] = item
+    if self.current < self.capacity - 1:
+      self.current += 1
+    else:
+      self.current = 0
 
   def get(self):
-    pass
\ No newline at end of file
+    filtered = filter(lambda x: x != None, self.storage)
```

Comment:
nice. I haven't seen this way before. You can also do it in one line:

return [x for x in self.storage if x is not None]